### ISSUE TITLE
"Effects" displays empty list and breaks the toolbar

### ISSUE THREAD

**Issue by 0xfadeef** [robaho/seashore#29]
Press "Effect Tools" > "Effect" in the toolbar, then press "Choose Effect" button. Effects list is empty and after that it's not possible to pick any other tool or use window buttons (to close the window, for instance).

OS X 10.9.5, app version: 2.1.8

**Comment by 0xfadeef** [comment]
@robaho Not only I deleted previous version of Seashore, I `rm`-ed everything found with `sudo find / -iname *seashore*`. Issue remains. But, by your advice, I checked Console and there is bunch of error messages of the same form for each plugin installed.
```
12/1/19 19:51:40,646	Seashore[30377]	Error loading /Users/sergeyfadeev/Applications/Seashore/Seashore.app/Contents/PlugIns/Brightness+.bundle/Contents/MacOS/Brightness+:  dlopen(/Users/sergeyfadeev/Applications/Seashore/Seashore.app/Contents/PlugIns/Brightness+.bundle/Contents/MacOS/Brightness+, 265): Library not loaded: /System/Library/Frameworks/CoreImage.framework/Versions/A/CoreImage
  Referenced from: /Users/sergeyfadeev/Applications/Seashore/Seashore.app/Contents/PlugIns/Brightness+.bundle/Contents/MacOS/Brightness+
  Reason: image not found
12/1/19 19:51:40,647	Seashore[30377]	unable to open bundle NSBundle </Users/sergeyfadeev/Applications/Seashore/Seashore.app/Contents/PlugIns/Brightness+.bundle> (not yet loaded)
 ```

**Comment by robaho** [comment]
@0xfadeef see this http://flummox-engineering.blogspot.com/2018/08/missing-coreimage-framework-in-macos-yosemite-10-10-5-quartz-framework.html

Not sure why it is being built that way, since I have compatibility set to 10.7 but I will investigate. Hopefully I can fix it for the next build.

**Comment by 0xfadeef** [comment]
@robaho I symlinked `CoreImage.framework` as it was suggested and looks like it resolves the problem, the app is working fine right now. Thank you for help.

**Comment by robaho** [comment]
@0xfadeef I'll probably ask you to remove the symlink and test with the next version - obviously doing the symlink is not ideal.

**Comment by robaho** [comment]
I removed any direct references to CoreImage in the projects in 2.1.9 - the built plugin bundles still seem to reference the direct path, but hopefully it works in the wild.

**Comment by 0xfadeef** [comment]
@robaho I unlinked `CoreImage.framework` and checked with version 2.1.9. Unfortunately, the issue still present. Please check the [screenshot](https://user-images.githubusercontent.com/30846958/51203407-04137300-1912-11e9-98f0-1b412e4e682e.png).

**Comment by robaho** [comment]
@0xfadeef Thanks for the update. I just downloaded an Apple sample core image application and built it. The resulting bundle had the same Core Image reference as Seashore - so it certainly appears the problem is with your system. I checked a bunch of other coreimage builds and problem reports around the web, and the framework should be there - a fresh OSX install may fix things.

Did you possibly downgrade OSX from a different (factory) installed version ?

Did previous versions of Seashore work, if so what was the last version you used successfully without the linkage ?

**Comment by robaho** [comment]
I am re-opening. I figured out the issue. The plugins were being required to compile for 10.14+, so when linking it assumed that is where the CoreImage framework was. After I switched to 10.7, they have different linkage instructions.

I am in the process of making a new build.

**Comment by robaho** [comment]
This should be fixed in version 2.2.0

**Comment by 0xfadeef** [comment]
@robaho So far not reproducible on `2.2.0`. Thank you!
